app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
```

Comment:
For importing multiple classes/modules, it's recommended to use parentheses. This makes the code cleaner and easier to read. Here's how you can do it:

python
from app.cafe import (
    Cafe,
)

from app.errors import (
    VaccineError,
    NotWearingMaskError,
)
```